app/src/main/java/org/literacyapp/analytics/dao/LetterLearningEventDao.java

```diff
@@ -0,0 +1,165 @@
+package org.literacyapp.analytics.dao;
+
+import android.database.Cursor;
+import android.database.sqlite.SQLiteStatement;
+
+import org.greenrobot.greendao.AbstractDao;
+import org.greenrobot.greendao.Property;
+import org.greenrobot.greendao.internal.DaoConfig;
+import org.greenrobot.greendao.database.Database;
+import org.greenrobot.greendao.database.DatabaseStatement;
+
+import java.util.Calendar;
+import org.literacyapp.analytics.dao.converter.CalendarConverter;
+
+import org.literacyapp.analytics.model.LetterLearningEvent;
+
+// THIS CODE IS GENERATED BY greenDAO, DO NOT EDIT.
+/** 
+ * DAO for table "LETTER_LEARNING_EVENT".
+*/
+public class LetterLearningEventDao extends AbstractDao<LetterLearningEvent, Long> {
+
+    public static final String TABLENAME = "LETTER_LEARNING_EVENT";
+
+    /**
+     * Properties of entity LetterLearningEvent.<br/>
+     * Can be used for QueryBuilder and for referencing column names.
+     */
+    public static class Properties {
+        public final static Property Id = new Property(0, Long.class, "id", true, "_id");
+        public final static Property DeviceId = new Property(1, String.class, "deviceId", false, "DEVICE_ID");
+        public final static Property Time = new Property(2, long.class, "time", false, "TIME");
+        public final static Property PackageName = new Property(3, String.class, "packageName", false, "PACKAGE_NAME");
+        public final static Property StudentId = new Property(4, String.class, "studentId", false, "STUDENT_ID");
+        public final static Property Letter = new Property(5, String.class, "letter", false, "LETTER");
+    }
+
+    private final CalendarConverter timeConverter = new CalendarConverter();
+
+    public LetterLearningEventDao(DaoConfig config) {
+        super(config);
+    }
+    
+    public LetterLearningEventDao(DaoConfig config, DaoSession daoSession) {
+        super(config, daoSession);
+    }
+
+    /** Creates the underlying database table. */
+    public static void createTable(Database db, boolean ifNotExists) {
+        String constraint = ifNotExists? "IF NOT EXISTS ": "";
+        db.execSQL("CREATE TABLE " + constraint + "\"LETTER_LEARNING_EVENT\" (" + //
+                "\"_id\" INTEGER PRIMARY KEY AUTOINCREMENT ," + // 0: id
+                "\"DEVICE_ID\" TEXT NOT NULL ," + // 1: deviceId
+                "\"TIME\" INTEGER NOT NULL ," + // 2: time
```

Comment:
@bikash We will add more detailed properties like duration later. Feel free to update the properties at https://github.com/literacyapp-org/literacyapp-model/tree/master/src/main/java/org/literacyapp/model/gson/analytics if you need to work with more types of data.